src/Test/Tasty/Plutarch/Property.hs

```diff
@@ -19,6 +19,9 @@ module Test.Tasty.Plutarch.Property (
 
     -- * Coverage-based properties
     classifiedProperty,
+
+    -- * Calssified property optimized for cases that always fails
+    alwaysFailProperty,
```

Comment:
I'd probably label this without mentioning 'classified properties' at all. It's not technically a term (universal and conditional properties are the terms we use), and it's not really 'classified' anyway.

Reply:
Yeah I forgot to change it :+1: